packages/core/src/http-client/http-client.ts

```diff
@@ -43,15 +43,19 @@ export async function buildHttpRequest(
 ): Promise<DestinationHttpRequestConfig> {
   if (customHeaders) {
     logger.warn(
-      `The custom headers are provided with the keys: ${Object.keys(
+      `The custom headers are provided with the keys: [ ${Object.keys(
         customHeaders
-      )}. These keys will overwrite the headers created by the SDK.`
+      ).join(
+        ', '
+      )} ]. These keys will overwrite the headers created by the SDK.`
```

Comment:
I'd prefer something like
```
`The custom headers are provided with the keys: 
  - ${Object.keys(
  customHeaders
).join(
  '\n  - '
)}
These keys will overwrite the headers created by the SDK.`
```
As it probably reads easier. But I can appreciate the easier copy paste with the `[]` but in that case it should also be double quoted like JSON...

Reply:
Done